pandas/core/window/rolling.py

```diff
@@ -1579,6 +1606,9 @@ def cov(
         ddof: int = 1,
         **kwargs,
     ):
+        if self.step not in [None, 1]:
+            raise NotImplementedError(f"invalid step: {self.step}")
```

Comment:
Maybe `"step not implemented for func"`

Reply:
I tested after adding `NotImplementedError` for `step` in non-fixed window indexers, and observed that `BaseWindow._apply_blockwise` swallows the error in its `except` statement. This statement was there already before my changes and surely for some reason, which I don't know, so I'm not sure how to fix this to let the error propagate.